packages/sdk/src/resolve/resolve-voucher-intercept.js

```diff
@@ -0,0 +1,10 @@
+import {get, isFn} from "../interaction/interaction.js"
+import {createSignableVoucher} from "./voucher.js"
+
+export async function resolveVoucherIntercept(ix) {
+  const fn = get(ix, "ix.voucher-intercept")
+  if (isFn(fn)) {
+    await fn(createSignableVoucher(ix))
```

Comment:
Do you need to do anything with this?

Reply:
not sure what you mean. this resolve function was renamed, and returns the voucher
it's used in resolve.js
https://github.com/onflow/fcl-js/blob/03bdf8342203281929c40abef5f67c1e892cfc53/packages/sdk/src/resolve/resolve.js#L62